ui/Screen/OrgRegistration.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+  import { pop } from "svelte-spa-router";
+
+  import {
+    IdentifierSelectionStep,
+    ModalLayout,
+    StepCounter,
+    TransactionSubmissionStep
+  } from "../DesignSystem/Component";
+  import { Title } from "../DesignSystem/Primitive";
+
+  const steps = {
+    PREPARE: 1,
+    SUBMIT: 2
+  };
+
+  let currentStep = steps.PREPARE;
+
+  const nextStep = () => currentStep++;
```

Comment:
It would be nice to move away from the current step pattern and rather push for a proper state machine for these multi-stage flows. Which could be implemented in ts and consumed here.